charts/node/values.yaml

```diff
@@ -102,8 +102,11 @@ node:
   serviceMonitor:
     enabled: false
     #namespace: monitoring
-    #interval: 10s
-    # scrapeTimeout: 10s
+    interval: 1m
+    scrapeTimeout: 30s
+    targetLabels:
+      - chain
+      - instance
```

Comment:
Should we really have this `instance` label by default. What does this correspond to ?

Reply:
Done!

Reply:
hmm, i think the reason behind that `instance` label was (though not targetted through) to have a common label for vm and kubernetes based deployments. would that be `node` then in that case?

Reply:
Yes node would be the new label. @lazam `instance` also need to be renamed to `node` here: https://github.com/paritytech/helm-charts/blob/main/charts/node/templates/service.yaml#L33